driver/config/config.go

```diff
@@ -69,6 +69,7 @@ const (
 	ViperKeyCourierSMTPURL                                   = "courier.smtp.connection_uri"
 	ViperKeyCourierSMTPClientCertPath                        = "courier.smtp.client_cert_path"
 	ViperKeyCourierSMTPClientKeyPath                         = "courier.smtp.client_key_path"
+	ViperKeyCourierEnableInvalidMessageDispatch              = "courier.enable_invalid_message_dispatch"
```

Comment:
Or maybe `courier.flows.recovery.enable_invalid_message` as this is really a feature of recovery. I'm just not very happy with this high-level, pretty undescriptive key.

Reply:
It's not just recovery, though. Verification as well. 

Ideally, this would be false by default, and an opt-in feature. So this key wouldn't be used except by people really wanting this feature. But that would be breaking.

Reply:
I think the default should be enabled - it prevents sending a ton of mails out when script kiddies play around with rate limiting. We can either make this configurable per flow (recovery, **login**, verification) or globally. My feeling is that someone will want to configure this for magic link login

For the concrete key name (regardless of where) I suggest:

```
notify_unknown_recipients = false|true
send_to_unknown_recipients = false|true
ignore_unknown_recipients = false|true
```

Reply:
"enabled" right now means, that the emails **will** be sent to unknown recipients. Is that what you meant? @aeneasr 

I like `courier.notify_unknown_recipients`. Maybe we can do:

```yaml
courier:
  notify_unknown_recipients:
    global: false|true
    verification: false|true
    recovery: false|true
    # login: false|true (in the future)
```

And we can set the default to `false` to prevent the rate-limit issues.

For OS this would be a breaking change, but for Ory Network, we can keep it enabled for existing projects, and set it to `false` for new projects (similar to the way we rolled out `code`), avoiding the breaki-ness. WDYT?

Reply:
Thinking about it a bit more, this should go to `selfservice.flows.<login|recovery|verificatio>` instead. The courier doesn't know what recovery is, it just sends mail :)

>And we can set the default to false to prevent the rate-limit issues.
>For OS this would be a breaking change, but for Ory Network, we can keep it enabled for existing projects, and set it to false for new projects (similar to the way we rolled out code), avoiding the breaki-ness. WDYT?

SGTM

Reply:
> Thinking about it a bit more, this should go to selfservice.flows.<login|recovery|verificatio> instead. The courier doesn't know what recovery is, it just sends mail :)

Makes sense to me.